### ISSUE TITLE
Unable to register a connection (v1.3.6)

### ISSUE THREAD

**Issue by spiralni** [wp-graphql/wp-graphql#1882]
wpgraphql version: Version 1.3.6

I am unable to register a connection. I've tried many ways and it's impossible.

I debug the code, and the method `register_graphql_connection` is executed successfully, but the connection is never registered .
I've tried changing the connection name, no luck either.

![image](https://user-images.githubusercontent.com/339101/116773837-6de9b500-aa15-11eb-8e56-1fe0b02145cb.png)

This is my code:

```
     function registerSpecialtyType() {
        $specialty_specs =  [
            'description' => 'Specialties',
            'fields' => [
                'id' => [
                    'type' => 'Int',
                    'description' =>  'ID',
                ],
                'name' => [
                    'type' => 'String',
                    'description' => 'Specialty name',
                ],
                'description' => [
                    'type' => 'String',
                    'description' => 'Description',
                ],
                'url' => [
                    'type' => 'String',
                    'description' => __( 'Picture URL', 'drnox-backend' ),
                ],
            ],
        ];
        register_graphql_object_type( 'Specialty', $specialty_specs );

        $config = [
            'fromType'         => 'RootQuery',
            'toType'           => 'Specialty',
            'fromFieldName'    => 'specialties',
            'connectionTypeName' => 'SpecialtiesConnection',
            'resolve' => function( $root, $args, $context, $info ) {
                $revolver = new SpecialtyConnectionResolver( $root, $args, $context, $info );
                return $revolver->get_connection();
            },
        ];
        register_graphql_connection( $config );
    }
  add_action( 'graphql_register_types', 'registerSpecialtyType'  );

```

Additional Info:

![image](https://user-images.githubusercontent.com/339101/116773944-3596a680-aa16-11eb-877b-e1513c2709a3.png)

**Comment by spiralni** [comment]
Rolling back to version 1.3.5 FIX the issue!

![image](https://user-images.githubusercontent.com/339101/116788152-576b4a00-aa65-11eb-9db8-ae387307e9d0.png)

**Comment by jasonbahl** [comment]
@spiralni I've been able to reproduce this on v1.3.6 but not 1.3.5, so it seems there's some sort of regression.

I'm looking into this and hope to have a resolution shortly.

**Comment by mitchierichie** [comment]
@jasonbahl @spiralni I got this on v1.3.6 until this issue is fixed, I was able to workaround it by registering my connection on plugins_loaded action, instead of the register_graphql_types action

**Comment by jasonbahl** [comment]
@mitchierichie thanks for posting the workaround. I do have this fixed in #1885 and I'm prepping a 1.3.7 release for it

**Comment by mitchierichie** [comment]
@jasonbahl I saw that, definitely appreciate the quick turnaround.
